src/HH4b/postprocessing/datacardHelpers.py

```diff
@@ -1,13 +1,17 @@
 from __future__ import annotations
 
 import logging
+import sys
 from dataclasses import dataclass, field
 
 import hist
 import numpy as np
-from hh_vars import years as all_years
 from hist import Hist
 
+sys.path.append("..")
```

Comment:
I'll try to fix this to be able to import from the installed HH4b package and test.

Reply:
Fixed I think, and I was able to remake the templates.